pkg/query/querier_test.go

```diff
@@ -482,9 +931,15 @@ func TestSortReplicaLabel(t *testing.T) {
 	}
 }
 
+const hackyStaleMarker = float64(-99999999)
```

Comment:
Could we please add a comment on _why_ this is needed? It's clear that NaN != NaN but it would be nice to know in the future quickly why this is needed.

Reply:
Just for testutil.Equals to work, but good point (: